apps/mobile-alphabet/src/app/Components/alphabet-card-detail.screen.tsx

```diff
@@ -80,84 +92,58 @@ export function AlphabetCardDetailScreen({ route }) {
         standalone_image,
     } = selectedCard;
 
+    const swipeRight = () => {
+        setSelectedLetterSequenceNumber(
+            ((selectedLetterSequenceNumber - 2 + alphabetCards.length) % alphabetCards.length) + 1
+        );
+    };
+
+    const swipeLeft = () => {
+        setSelectedLetterSequenceNumber((selectedLetterSequenceNumber % alphabetCards.length) + 1);
```

Comment:
Are you sure this works? l am a bit confused as to whether start indexing at 1 or 0. Let's review this logic together before merging. I'm probably just missing some context here.

Reply:
We refactored this.